docfx_project/ziti/overview.md

```diff
@@ -5,9 +5,71 @@ uid: zitiOverview
 # Overview
 
 ## Why Ziti?
-Ziti represents the next generation of secure networking for your applications and has arrived with the Ziti platform,
-created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust
-entry point into your network or as a secure zero-trust proxy to other hosted services.
+Ziti represents the next generation of secure, open-source networking for your applications. Ziti has several components.
+
+* The Ziti fabric provides a scalable, plugable, networking mesh with built in smart routing
+* The Ziti edge components provide a secure, Zero Trust entry point into your network
+* The Ziti SDKs allow you to integrate Ziti directly into your applications
+* The Ziti tunnelers and proxies allow existing applications and networks to take advantage of a Ziti deployment 
+
+**Security Features**
```

Comment:
At some point, not saying this round, we should highlight Ziti's ability to name services beyond DNS and IP - and how that it in some cases allows you to simplify complex networking that has IP management issues (collision/overlap).

** Service Configuration Features**
  * Reduce service connections addresses to Ziti service names
  * Abstract away complex IP management configurations
  * Supersede traditional DNS with your own Ziti network level hostnames/top level domains